packages/phrases/src/locales/de/errors/password.ts

```diff
@@ -1,7 +1,7 @@
 const password = {
   unsupported_encryption_method: 'Die Verschlüsselungsmethode {{name}} wird nicht unterstützt.',
   pepper_not_found: 'Password pepper not found. Please check your core envs.',
-  password_rejected: 'Password rejected. Please check if your password meets the requirements.', // UNTRANSLATED
+  rejected: 'Password rejected. Please check your password meets the requirements.', // UNTRANSLATED
```

Comment:
```suggestion
  rejected: 'Password rejected. Please check if your password meets the requirements.', // UNTRANSLATED
```

Reply:
Why dropping the `if`?

Reply:
a mistake, will update in a following PR